prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -240,4 +288,19 @@ class LivdTableDownload : CliktCommand(
         TermUi.echo("\tSUCCESS: The $outputfile file is created.\n")
         return true
     }
+
+    /**
+     * Updates the LIVD lookup table name [livdLookupTable] of CSV file.  It setups PRIME CLI Lookup Table Create
+     * Command line options.  And then, it calls the create lookup table command to create the new version of lookup
+     * table.  Note, it always create the new version regardless since it uses -f option.
+     */
+    private fun updateTheLivdLookupTable(livdLookupTable: File) {
+        val tableName = livdLookupTable.nameWithoutExtension
+        TermUi.echo("Creating table $tableName...")
+        val args: MutableList<String> = mutableListOf(
+            "-e", environment.toString().lowercase(), "-n", tableName,
+            "-i", livdLookupTable.absolutePath, "-s", "-a", "-f"
```

Comment:
Remove the silent, or add a new options to make it optional.  This removes the ability to see the diff, which I think it will be useful to see that the new changes are.  Also, do not use -f.  We do not want to create duplicate tables.  I think even the -a needs to be removed to allow the user to activate the table when appropriate.

I tested it without those options and got this message:
" Skipping New Lookup Table LIVD-SARS-CoV-2 is conflicting with the existing table version: 2"
Can you please change the word conflicting to a duplicate?

The diff that the tool shows is so broken with the LIVD table.  That would be another issue.

Reply:
Done.